src/components/grid-aware/ProgramBlock/ProgramBlock.module.css

```
@@ -0,0 +1,139 @@
+/* Background bleed */
+
+.bleedWrapper {
+  display: grid;
+  grid-template-columns: var(--bleed-grid-template-columns);
+  margin: auto;
+  margin-bottom: 100px;
+  max-width: var(--bleed-max-width);
+}
+
+/* Override grid item default of auto so that item contents cannot stretch the grid itself */
+.bleedWrapper > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .bleedWrapper {
+    grid-template-columns: 40px 1fr 40px;
+    margin-bottom: 80px;
+  }
+}
+
+.bleedMainContent {
+  grid-column: 2;
+}
+
+
+/* Main grid */
+
+.gridParent {
+  column-gap: var(--grid-column-gap);
+  display: grid;
+  grid-template-columns: repeat(var(--grid-column-count), minmax(0, 1fr));
+}
+
+.gridParent > * {
+  min-width: 0;
+}
+
+@media (--tablet-and-down) {
+  .gridParent {
+    display: flex;
+    flex-direction: column-reverse;
+  }
+}
+
+
+/* Main grid areas */
+
+.gridAreaText {
+  grid-column: 6/ span 6;
```

Nit: Let's put a space between the `6` and the `/`.

```suggestion
  grid-column: 6 / span 6;
```